src/integrationTest/java/uk/gov/hmcts/reform/professionalapi/FindOrganisationsByPbaStatusTest.java

```diff
@@ -0,0 +1,105 @@
+
+@SuppressWarnings("unchecked")
+public class FindOrganisationsByPbaStatusTest extends AuthorizationEnabledIntegrationTest {
+
+    @Test
+    public void get_request_returns_organisations_with_correct_pba_status() throws JsonProcessingException {
+        createOrganizationsAndEditPbas();
+
+        List<OrganisationsWithPbaStatusResponse> orgPbaResponse = (List<OrganisationsWithPbaStatusResponse>)
+                professionalReferenceDataClient.findOrganisationsByPbaStatus(
+                        PbaStatus.ACCEPTED.toString(), hmctsAdmin, Boolean.FALSE);
+
+        assertThat(orgPbaResponse).isNotNull();
+    }
+
+    @Test
+    public void get_request_returns_no_organisations_for_given_pba_status() throws JsonProcessingException {
+        createOrganizationsAndEditPbas();
+
+        List<OrganisationsWithPbaStatusResponse> orgPbaResponse = (List<OrganisationsWithPbaStatusResponse>)
+                professionalReferenceDataClient.findOrganisationsByPbaStatus(
+                        PbaStatus.REJECTED.toString(), hmctsAdmin, Boolean.FALSE);
+
+        assertThat(orgPbaResponse).isEmpty();
+    }
+
+    @Test
+    public void get_request_returns_400_when_invalid_pba_status() throws JsonProcessingException {
+        createOrganizationsAndEditPbas();
+
+        Map<String, Object> errorResponseMap = (Map<String, Object>) professionalReferenceDataClient
+                .findOrganisationsByPbaStatus("Invalid Status", hmctsAdmin, Boolean.FALSE);
+        ErrorResponse errorResponse = (ErrorResponse) errorResponseMap.get("response_body");
+
+        assertThat(errorResponseMap).isNotNull().containsEntry("http_status", HttpStatus.BAD_REQUEST);
+        assertThat(errorResponse.getErrorMessage()).isEqualTo(ErrorConstants.INVALID_REQUEST.getErrorMessage());
+    }
+
+    @Test
+    public void get_request_returns_403_when_invalid_role() throws JsonProcessingException {
+        createOrganizationsAndEditPbas();
+
+        Map<String, Object> errorResponseMap = (Map<String, Object>) professionalReferenceDataClient
+                .findOrganisationsByPbaStatus(PbaStatus.ACCEPTED.toString(), "invalid role", Boolean.FALSE);
+        ErrorResponse errorResponse = (ErrorResponse) errorResponseMap.get("response_body");
+
+        assertThat(errorResponseMap).isNotNull().containsEntry("http_status", HttpStatus.FORBIDDEN);
+        assertThat(errorResponse.getErrorMessage()).isEqualTo(ErrorConstants.ACCESS_EXCEPTION.getErrorMessage());
+    }
+
+    @Test
+    public void get_request_returns_401_when_invalid_role() throws JsonProcessingException {
+        Map<String, Object> errorResponseMap = (Map<String, Object>) professionalReferenceDataClient
+                .findOrganisationsByPbaStatus(PbaStatus.ACCEPTED.toString(), hmctsAdmin, Boolean.TRUE);
+
+        assertThat(errorResponseMap).isNotNull().containsEntry("http_status", HttpStatus.UNAUTHORIZED);
+    }
+
+
+    private void createOrganizationsAndEditPbas() {
```

Comment:
Are the below not a valid use cases?
- Get the list of organisations by pba status when status = ACCEPTED/REJECTED. You have only created ACTIVE companies in the test data and verifying that the response is empty.
- When null is passed as the pba status